climate_scanner/coarse_article_classifier/data_utils.py

```diff
@@ -26,4 +43,82 @@ def data_cleaning(self, text):
         while ensuring least possible loss of information
         """
         pass
+
+    def _remove_hyperlinks(self, text):
+        """ This function removes hyperlinks from texts
+            inputs:
+            - text """ 
+        text = text.apply(lambda x: re.sub(r"http\S|www\.\S++", " ", x))
```

Comment:
Are theses apply(lambda... wrappers necessary here? I think you should be able to just apply the functions directly.

Reply:
in fact, you can replace all of them and use just one apply function. Please refer to the data_processing step in the trends_innovation_classifier.